docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +68,347 @@ To enable your application to use the image util functionality:
    ret = image_util_decode_destroy(decode_h);
+   ```
 
-   ret = media_format_create(&fmt);
-   ret = media_format_set_video_mime(fmt, colorspace);
-   ret = media_format_set_video_width(fmt, width);
-   ret = media_format_set_video_height(fmt, height);
+<a name="image_colorspace"></a>
+## Converting Image Color Space
 
-   ret = media_packet_create_alloc(fmt, NULL, NULL, &src);
-   ret = media_packet_get_buffer_size(src, &size);
+To convert an image from one color space of an image to another:
 
-   src = malloc(size);
-   ret = media_packet_get_buffer_data_ptr(src, &src_ptr);
-   ret = memcpy(src_ptr, img_source, size);
+1. Create a transformation handle using `image_util_transform_create()`:
+
+   ```
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-<a name="colorspace"></a>
-## Converting the Color Space
+2. Set the target color space using `image_util_transform_set_colorspace()`:
 
-To convert an image from one color space to another:
+   ```
+   ret = image_util_transform_set_colorspace(transform_h, colorspace);
+   ```
 
-1. Create a transformation handle using the `image_util_transform_create()` function:
+3. Execute the transformation using `image_util_transform_run2()`:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   ret = image_util_transform_run2(transform_h, src_image, &dst_image);
    ```
 
-2. Set the target color space using the `image_util_transform_set_colorspace()` function:
+4. Execute the transformation using `image_util_transform_run2_async()`:
 
    ```
-   ret = image_util_transform_set_colorspace(handle, colorspace);
+   ret = image_util_transform_run2_async(transform_h, src_image,
+                                        (image_util_transform_completed2_cb)completed_callback,
+                                        user_data);
    ```
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+   > **Note**
+   >
+   > `image_util_transform_run2()` and `image_util_transform_run2_async()` only converts the color space. These functions do not change the image width, height, or any other image property. Due to these restrictions of the image processing library, not all color space combinations are supported for conversion.
+
+5. Handle the transformation results in `image_util_transform_completed2_cb()`, which is invoked after the transformation is complete.
+
+6. After the transformation is complete, destroy the transformation handle using `image_util_transform_destroy()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_destroy(transform_h);
+   ```
+
+<a name="mediapacket_colorspace"></a>
+## Converting Media Packet Color Space
+
+To convert an image from one color space of a media packet to another:
```

Comment:
Because the feature is converting the colorspace of an image, I think that the sentence should be below.
Could you please check?

To convert one color space of a media packet to another:
To convert one color space of a media packet to another (color space)

Reply:
@jiyong2, thank you for the response. As per me, 
To convert one color space of a media packet to another: 
is better.